docs/dev_101_services.md

```diff
@@ -59,20 +59,33 @@ Services are published under the domain name of your integration, so in `service
 ```yaml
 # Example services.yaml entry
 
+# Service ID
 set_speed:
+  # Service name as shown in UI
+  name: Set fan speed.
```

Comment:
This is shown in a box, prefixed with the integration name (fan in this case).
```suggestion
  name: Set speed
```

Results in: `Fan: Set speed`